internal/controller/volsync/vshandler.go

```diff
@@ -361,6 +366,8 @@ func (v *VSHandler) validatePVCBeforeRS(rsSpec ramendrv1alpha1.VolSyncReplicatio
 		}
 
 		if pvcIsMounted {
+			v.workloadStatus = "active"
```

Comment:
At this point VRG is Primary, so this value of workloadStatus is currently unused